app/static/app/js/components/Map.jsx

```diff
@@ -139,7 +141,7 @@ class Map extends React.Component {
                                     <i class="fa fa-cube"></i> 3D
                                 </button>`;
 
-            layer.bindPopup(popup);
+            //layer.bindPopup(popup);
```

Comment:
Is this disabling the popup permanently?

Reply:
Maybe, we could find a way to disable this popup when the user start drawing the polygon for volume estimation!

Reply:
I think that would be a good way to do it, yes.